do no even bother looking at the current node version

Index: src/setup_node_env/node_version_validator.js
--- src/setup_node_env/node_version_validator.js.orig
+++ src/setup_node_env/node_version_validator.js
@@ -186,6 +186,8 @@ function versionCompare(versionA, versionB) {
   return -1;
 }
 
+if (false) {
+
 var currentVersion = process && process.version;
 
 if (!currentVersion) {
@@ -220,4 +222,5 @@ if (satisfiesBottom && rangeTop) {
 if (!satisfiesBottom || rangeTop && !satisfiesTop) {
   console.error('OpenSearch Dashboards cannot start up using the Node.js runtime v' + version.major + '.' + version.minor + '.' + version.patch + '. ' + getVersionCompatibilityMessage());
   process.exit(21);
-}
\ No newline at end of file
+}
+}
